examples/step-52/step-52.cc

```diff
@@ -449,17 +449,32 @@ namespace Step52
     DataOut<2> data_out;
 
     data_out.attach_dof_handler(dof_handler);
-    data_out.add_data_vector(solution, "solution");
+    std::string solution_name = "solution_" + method_name;
+    data_out.add_data_vector(solution, solution_name);
 
     data_out.build_patches();
 
     data_out.set_flags(DataOutBase::VtkFlags(time, time_step));
 
-    const std::string filename = "solution-" + method_name + "-" +
-                                 Utilities::int_to_string(time_step, 3) +
-                                 ".vtu";
+    const std::string filename =
+      solution_name + Utilities::int_to_string(time_step, 3) + ".vtu";
```

Comment:
you dropped the `"-"` between the method name and the number of the time step here. I think it would be useful to keep it.